src/OpenProject.Revit/Entry/ConfigurationLoader.cs

```diff
@@ -37,27 +39,20 @@ public static string GetBcfierWinExecutablePath()
     private static string GetConfigurationFilePath()
     {
       var configPath = Path.Combine(
-        Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData),
-        "OpenProject.Revit",
+        ConfigurationConstant.OpenProjectApplicationData,
         "OpenProject.Revit.Configuration.json");
 
-      if (!File.Exists(configPath))
-      {
-        // If the file doesn't yet exist, the default one is created
-        using (var configStream = typeof(ConfigurationLoader).Assembly.GetManifestResourceStream("OpenProject.Revit.OpenProject.Revit.Configuration.json"))
-        {
-          var configDirName = Path.GetDirectoryName(configPath);
-          if (!Directory.Exists(configDirName))
-          {
-            Directory.CreateDirectory(configDirName);
-          }
+      if (File.Exists(configPath)) return configPath;
 
-          using (var fs = File.Create(configPath))
-          {
-            configStream.CopyTo(fs);
-          }
-        }
-      }
+      // If the file doesn't yet exist, the default one is created
+      using Stream configStream =
+        typeof(ConfigurationLoader).Assembly.GetManifestResourceStream(
+          "OpenProject.Revit.OpenProject.Revit.Configuration.json");
+      var configDirName = Path.GetDirectoryName(configPath);
+      if (!Directory.Exists(configDirName)) Directory.CreateDirectory(configDirName);
```

Comment:
You are not a fan of curly braces😀

Reply:
Got me there :D But, I've of course a very subjective preferred style, and whenever you or anyone else say, something is not really comprehensible or too complex for a single line or a missing body, I'll always consider a change ;)

Reply:
No it's fine for me, I don't want to enforce my style. Especially since you're doing the coding, so you should be free to follow your style😀

But I'll always point this out😀